src/pages/public/ForgotPassword.tsx

```diff
@@ -0,0 +1,58 @@
+import { Form, Formik, FormikProps } from "formik";
+import React from "react";
+import { PrimaryButton } from "~/components/Button";
+import { InputField } from "~/components/Form";
+
+const initialValues = {
+  email: ""
+};
+
+const validate = (values) => {
+  const errors: { email?: string; password?: string } = {};
+
+  if (!values.email) {
+    errors.email = "Pflichtfeld";
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
+    errors.email = "Ungültige E-Mail-Addresse";
+  }
+
+  return errors;
+};
```

Comment:
Wir werden ja einige Forms haben. Wollen wir daher vllt direkt zu Beginn auf [**yup**](https://github.com/jquense/yup) setzen. Validation in Formik läuft dann stattdessen über die [validationSchema](https://jaredpalmer.com/formik/docs/guides/validation#validationschema) prop.

Reply:
Wie würdest du das mit dem import machen? import * as yup? Oder die einzelnen Objekte?
Fand halt import {string, object} from 'yup' und dann einfach object() im Code aufzurufen irgendwie nicht aussagekräftig und import {object as yupObject} ist auch irgendwie geht so geil

Reply:
Habe tatsächlich immer `import * as yup`, was natürlich die ganze Library importiert - und die ist schon recht groß.